scanpy/tools/_louvain.py

```diff
@@ -133,7 +136,24 @@ def louvain(
                 g, partition_type,
                 **partition_kwargs,
             )
-            # adata.uns['louvain_quality'] = part.quality()
+            if 'quality' in dir(part):
+                q = part.quality()
+                if isinstance(part, louvain.RBConfigurationVertexPartition) and q > 1.0: #scale
+                    m = weights.sum() if use_weights else g.ecount()
+                    q /= m if directed else m*2
+                if isinstance(part, (
+                    louvain.RBConfigurationVertexPartition,
+                    louvain.ModularityVertexPartition),
+                ):
```

Comment:
Hmm, not quite. The idea is that you can add or remove single lines without touching other lines (e.g. for nicer git diffs). So you want the lines that have the same semantic meaning (one entry in a tuple) to look exactly the same.

```py
if isinstance(part, (
    louvain.RBConfigurationVertexPartition,
    louvain.ModularityVertexPartition,
)):
```

Reply:
Ah ok, sorry. I liked the `)):` part actually :)

Reply:
Thank you!